sdk/nodejs/storage/account.ts

```diff
@@ -430,7 +435,7 @@ export interface AccountState {
     /**
      * Defines the access tier for `BlobStorage` and `StorageV2` accounts. Valid options are `Hot` and `Cool`, defaults to `Hot`.
      */
-    readonly accessTier?: pulumi.Input<string>;
+    readonly accessTier?: pulumi.Input<string | AccessTier>;
```

Comment:
Wouldn't we be able to make it simply `pulumi.Input<AccessTier>` (applies to all types)? I think these values are less ambiguous than locations, and we may just enforce the proper values on users?

Reply:
I just didn't want to make a breaking change

Reply:
I think we discussed with @jen20 that this kind of change might be fine, let's wait for his take